libraries/AP_Scripting/applets/leds_on_a_switch.lua

```diff
@@ -0,0 +1,44 @@
+-- leds_on_a_switch.lua: control led brightness with a radio switch
+--
+
+local rc_switch = rc:find_channel_for_option(300)  --AUX FUNC sw for mission restart
+
+if not rc_switch then  -- requires the switch to be assigned in order to run script
+    gcs:send_text(MAV_SEVERITY.INFO, "LED AUX channel not assigned")
+    return
+end
+
+-- bind a parameter to a variable
+function bind_param(name)
+    local p = Parameter()
+    assert(p:init(name), string.format('could not find %s parameter', name))
+    return p
+end
+
+-- constants
+local AuxSwitchPos = {LOW=0, MIDDLE=1, HIGH=2}
+local NTF_LED_BRIGHT        = bind_param("NTF_LED_BRIGHT")
```

Comment:
If your happy for this to only work on recent versions 4.4 and later you can remove the `bind_param` helper and just do this:
```suggestion
local NTF_LED_BRIGHT = Parameter("NTF_LED_BRIGHT")
```